worldengine/simulations/precipitation.py

```diff
@@ -1,6 +1,7 @@
 import random
 import time
 import numpy
+import math
```

Comment:
I'm fairly certain I didn't spot a use of this module. (If you were using it indeed, you would probably find an equivalent function in numpy anyway.)